bank-accounts.rb

```diff
@@ -51,22 +83,43 @@ def initialize(id, initial_balance, open_date = nil, owner = nil)
         @date = DateTime.strptime(open_date, "%Y-%m-%d %H:%M:%S %z")
       end
       @owner = owner
+      @colors_array = [:red, :light_red, :green, :light_green, :yellow, :light_yellow, :blue, :light_blue, :magenta, :light_magenta, :cyan, :light_cyan]
```

Comment:
It seems like this would be good to set as a constant rather than an instance variable since I don't think the colors array will change